scripts/gui/microbe_editor.mjs

```diff
@@ -21,6 +21,21 @@ let limitMovesPerSession = true;
 // The full patch data
 let patchData = null;
 
+// Object that store actual patch dataset// Save data of actual patch
+const actualPatchData = {
```

Comment:
All the patch data is already stored in `patchData` and the selected patch and current patch variables. I don't think introducing this new global variable is necessary.